freebsd/FreeBSDProcessList.c

```diff
@@ -109,8 +109,8 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* dynamicMeters, H
 
    size_t sizeof_cp_time_array = sizeof(unsigned long) * CPUSTATES;
    len = 2; sysctlnametomib("kern.cp_time", MIB_kern_cp_time, &len);
-   fpl->cp_time_o = xCalloc(cpus, sizeof_cp_time_array);
-   fpl->cp_time_n = xCalloc(cpus, sizeof_cp_time_array);
+   fpl->cp_time_o = xCalloc(1, sizeof_cp_time_array);
+   fpl->cp_time_n = xCalloc(1, sizeof_cp_time_array);
```

Comment:
Given this allocates an array of `unsigned long` this should better be:
```suggestion
   fpl->cp_time_o = xCalloc(CPU_STATES, sizeof(unsigned long));
   fpl->cp_time_n = xCalloc(CPU_STATES, sizeof(unsigned long));
```

Reply:
There is a somewhat subtle issue here: `calloc` does not only an overflow check that the current version is missing, but also ensures proper alignment. Depending on the exact values used here, this [may make a difference](https://stackoverflow.com/a/501949), causing the actual allocation being larger due to proper alignment of allocated elements (See note [here](https://en.cppreference.com/w/c/memory/calloc)).

Reply:
`CPU_STATES` -> `CPUSTATES` in the suggested change

Reply:
Thanks for the extra explanations @BenBE! Totally makes sense, I updated.

@fasterit: Thanks for the correction.